draft-ietf-rats-corim.md

```diff
@@ -1823,7 +1823,7 @@ Any CoRIM that has been secured by a cryptographic mechanism, such as a signatur
 Other selection criteria MAY be applied.
 For example, if the Evidence format is known in advance, CoRIMs using a profile that is not understood by a Verifier can be readily discarded.
 
-The selection process MUST yield at least one usable tag.
+The selection process MUST yield at least one usable tag else the Verifier MUST NOT proceed to the next stage.
```

Comment:
I think this requirement should be removed actually. There are cases where you verify evidence purely through policy and no corims. You just trivially go from phase 1 to phase 5

Reply:
>and no corims.

Since this directive is in the CoRIM specification it's valid to assume there's a CoRIM.

Reply:
For the theory of operation, I still don’t think halting is right. You can be prepared for corims and have none available while still wanting to advance in the appraisal pipeline